FrEIA/modules/splines/rational_quadratic.py

```diff
@@ -164,7 +164,7 @@ def rational_quadratic_spline(x: torch.Tensor,
 
         # Eq 29 in the appendix of the paper
         discriminant = b ** 2 - 4 * a * c
-        assert torch.all(discriminant >= 0)
+        assert torch.all(discriminant >= 0), f"Discriminant must be positive, but is violated by {torch.min(discriminant)}"
```

Comment:
This was an assert, because I assumed it could never be violated. If we want error messages, we should probably `raise` and Exception.

Reply:
Fair point. This was actually just a diagnostics tool for me, to see what the cause of model the assertion was. But since the intended purpose of the assert it was to catch negative discriminants how about changing the line to:
```Python
assert not torch.any(discriminant < 0)
```
If the training diverges to NaN this should result in an error elsewhere.

Reply:
> If the training diverges to NaN this should result in an error elsewhere.

While this is true, I think the current assertion is better, because it ensures the remaining algorithm is computable.

Reply:
Agreed. I have changed it to a RuntimeError